src/extra/libs/freetype/lv_freetype.c

```diff
@@ -151,6 +154,11 @@ void lv_ft_font_destroy(lv_font_t * font)
 #endif
 }
 
+bool lv_ft_font_check_type(const lv_font_t * font)
```

Comment:
What does `type` mean here? It seems like a "validity" test.

Reply:
My idea is to check if this is a lv_freetype font, or a built-in font. If we have some new font implementation, creating such function can help developers to use different implementation for different fonts.

Reply:
I see. 

Can this function be `static`?

What about a  name like `is_ft_font`?

Reply:
I'm afraid this function can't be static. It compares function pointers with static functions inside lv_freetype.c, so itself has to be global.
Since it's a global function, the name convention should better follow other functions.

Reply:
By the way it's not used currently. Maybe I can remove it and aff when we really need it.